projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/datajobs/DeploymentModelConverter.java

```diff
@@ -37,26 +38,65 @@ public static JobDeployment toJobDeployment(
     return deployment;
   }
 
-  public static JobDeployment toJobDeployment(
-      String teamName, String jobName, DataJobDeployment jobDeploymentStatus) {
-    JobDeployment deployment = new JobDeployment();
-    deployment.setDataJobTeam(teamName);
-    deployment.setDataJobName(jobName);
+  public static DesiredDataJobDeployment toDesiredDataJobDeployment(
+      JobDeployment jobDeploymentStatus) {
```

Comment:
```suggestion
      JobDeployment jobDeployment) {
```

Let's name the variable more appropriately. It confused me for a bit. I thought the type is JobDeploymentStatus